src/ansys/mechanical/core/embedding/ui.py

```diff
@@ -22,7 +22,10 @@
 """Run Mechanical UI from Python."""
 
 from pathlib import Path
-from subprocess import Popen
+
+# Subprocess is needed to launch the GUI and clean up the process on close.
+# Excluding bandit check.
+from subprocess import Popen  # nosec
```

Comment:
You need to add the specific bandit code you want to ignore: ``# nosec: B404`` in this case